test_data/gbif/init_test_data.sql

```diff
@@ -0,0 +1,6 @@
+CREATE EXTENSION IF NOT EXISTS postgis WITH SCHEMA public;
```

Comment:
why with schema public? we don't need it there

Reply:
Maybe you have an idea, but when I install it into schema gbif and then try to install it again with schema abcd, it says it is already installed.
When I install it into public, it can be used anywhere.